### PR TITLE
Feature: intuit sign-up page

### PR DESCRIPTION
**Pull request by StarSheriff2**
We have cloned the signup page of Intuit.com using HTML and CSS only. The page is not functional.

The main goal of this project was to work with the form element and all its parts.

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+  font-size: 10px;
+}
+
+.buttons {
+  background-color: #0077c5;
+  border: 1px solid #4c4c4c;
+  border-radius: 5px;
+  color: white;
+  /*! font-size: 16px; */
+  box-shadow: 2px 2px 3px rgba(180, 180, 180, 40);
```
Comment (Spayco) [inline css/styles.css:38]: - [ ] delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))  
:x: **commented piece of codes** /\* `.nav { color: red;}` \*/  
:heavy_check_mark: **comments about code** /\*Navigation \*/

File: css/styles.css [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+  padding: 0 8px;
+}
+
+/* header and sign in button*/
+
+header {
+  position: absolute;
+  right: 15px;
+  /*! margin-right: 10px; */
+}
```
Comment (Spayco) [inline css/styles.css:56]: - [ ] delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))  
:x: **commented piece of codes** /\* `.nav { color: red;}` \*/  
:heavy_check_mark: **comments about code** /\*Navigation \*/

File: index.html [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+        <input type="tel" id="phone" name="phone">
+        <span class="font-small phone-rates">Standard call, messaging or data rates may apply.</span>
+
+        <label for="password">Password</label>
+        <input type="text" id="password" name="password" placeholder="&#xf3c1">
+
+        <button class="buttons submit-btn" type="submit">
+          <i class="fas fa-lock"></i>
+           Create account
+        </button>
```
Comment (Spayco) [inline index.html:52]: - [ ] the button should be centered and wider ([link to page](https://accounts.intuit.com/signup.html))

File: README.md [hunk 4]
```diff
@@ -1 +1,36 @@
...
+In this project we cloned the sign-up page of intuit using HTML and CSS. The purpose of this project was to use the form elements in HTML.
+
+## Built With
+
+- HTML and CSS
+
+## Live Demo
+
+[Live Demo Link](https://starsheriff2.github.io/Youtube-Clone/)
+
```
Comment (Spayco) [inline README.md:18]: - [ ] your live demo link is for another project.

File: README.md [hunk 5]
```diff
@@ -1 +1,36 @@
...
+## Authors
+
+👤 **Arturo Alvarez**
+
+- GitHub: [@githubhandle](https://github.com/StarSherif12)
+
+
+👤 **Thales Neves**
+
+- GitHub: [@githubhandle](https://github.com/thneves)
```
Comment (Spayco) [inline README.md:28]: - [ ] Add more links, everyone can know what's your GitHub link by clicking on your username.

**Review by Spayco — CHANGES_REQUESTED** [review]
# Changes Requested 

Hello Team! 👋 
Awesome job so far!! 👍 you still need to make some changes though! 

## Required Changes ♻️ 

- [ ] Fix your linting errors, these errors are occurring because of the HTML file inside your images folder.

* [ ] Files not related to the project should not be committed and added to .gitignore (e.g. `node_modules`) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/gitignore.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
Delete the HTML and js files (and its folder) inside your images folder, you don't need them and they are not part of the project.

## Optional 


* [ ] use the [`<nav>` tag](http://html5doctor.com/nav-element/) for the major navigation links





### please make the appropriate changes and ask for another code review ✔️ 

Happy Coding!

Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

File: index.html (hunk 3 above)
Comment (thneves) [reply index.html:52]: Our button is wider as the input fields exactly like in the original page

**Review by kevincappuccino — APPROVED** [review]
# [STATUS] PROJECT APPROVED 🍾 (⚠)

### I really like your project! Congrats team, you guys did an excellent job, you have my approval to go ahead and merge this pull request!

_**Clean and modular code, following the best practices and the design a looks great, keep it up!**_ 🥇

**⚠These are mandatory changes, but since It wasn't pointed out earlier, I'll mark them as [OPTIONAL]:**
- [x] The password input should be `type="password"` instead of `type="text"`

## As always, I wish you all the very best and see you both at the top🚀


👨🏻‍💻 **TSE: Kevin Alves**

  

📫 Have any question? Better call Kevin:

<a  href="mailto:kevinalves.js@gmail.com"> gmail</a> [Twitter Follow](https://twitter.com/intent/follow?screen_name=kevinnnmn) [](https://microverse-students.slack.com/team/UT86D3AP4)

**Pull request merged**